index.html

```diff
@@ -79,6 +79,245 @@ <h1>
           </div>
         </section>
         <!-- Home-page section Ends here -->
+
+        <!-- Work-section Starts here -->
+
+        <section id="work-section">
+          <div class="grid-container">
+            <!-- portfolio - 1 -->
+            <div class="card">
+              <div class="card-content">
+                <a href="#"
+                  ><img src="./images/Portfolio 1.png" alt="portfolio-1"
+                /></a>
+                <a href="#"><h1>TONIC</h1></a>
+                <div class="work-details">
+                  <a href="#"><h3>CANOPY</h3></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>Back End Dev</h4></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>2015</h4></a>
+                </div>
+                <p>
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul>
+                  <li class="work-tools">
+                    <button type="button" class="tools">html</button>
+                    <button type="button" class="tools">css</button>
+                    <button type="button" class="tools">javascript</button>
+                  </li>
+                </ul>
+                <div>
+                  <button type="button" class="primary-btn">See Project</button>
+                </div>
```

Comment:
- [OPTIONAL] Kindly, make sure the button interactions (enable, hover, etc.) works as expected 
on `See Project` buttons 💯👍